posix.mak

```diff
@@ -434,3 +434,13 @@ ${STABLE_DMD}:
 
 ${DUB}: ${DUB_DIR}/.cloned ${STABLE_DMD}
 	cd ${DUB_DIR} && DC=$(abspath ${STABLE_DMD}) ./build.sh
+
+################################################################################
+# Dman tags
+################################################################################
+
+chmgen : chmgen.d $(DMD)
+	$(DMD) -I${PHOBOS_DIR} -g chmgen.d
+
+d.tag : chmgen $(ALL_FILES)
```

Comment:
I'm having a race condition because this target depends on the druntime/phobos documentation without stating so.
Would be fine to just add druntime-release and phobos-release as dependencies, or does it also depend on the prerelease stuff?